src/coreclr/src/System.Private.CoreLib/src/Internal/Runtime/InteropServices/ComponentActivator.cs

```diff
@@ -39,6 +39,31 @@ private static string MarshalToString(IntPtr arg, string argName)
             return result;
```

Comment:
The check for null in MarshalToString is redundant. You can delete one of them while you are on it.

Reply:
It was probably added so the compiler knows we can't return null. Is adding '!' preferred over this?

Reply:
`!` is preferred over adding unreachable code.